### PR TITLE
Focus failures

### LINKED ISSUE #306: Focus Visible Failures
**Describe the bug**
Keyboard users who use the tab key to navigate through page aren't able to see which dataset they are focused on on the page. The elements ARE focused but not visibly; updated to include outline for visual tracking.

**To Reproduce**
1. Go to: https://scout.tsdataclinic.com/explore/NYC
2. Advance through screen using Tab key to 1st dataset link.
3. Unable to see where I was on the page as there's no outline to indicate focus.
4. Added outline to dataset link to allow for visual tracking of focused element.

**Expected behavior**
User should be able to see where their tab focus is when navigating through the page.

**Screenshots**

Before Dataset Link Outline

Dataset Link With Outline

**Additional context**

### PR DESCRIPTION
**Pull request by Detroit-the-Dev**
## Summary

Please include a summary of the changes. Also include relevant motivation and context. List any dependencies that are required for this change.

Keyboard users who use the tab key to navigate through page aren't able to see which dataset they are focused on on the page. The elements ARE focused but not visibly; updated to include outline for visual tracking.

## Screenshots or Videos (if applicable)

Include screenshots or video recordings to show off your change (if applicable).

Attached screenshots of dataset links, before AND after outline added.
Dataset Link With Outline
Before Dataset Link Outline

## Related Issues

List the related issues to this change. For example, `Closes #xyz`

Closes #306 

## Test Plan

Please describe the exact steps you followed to test your change. Be as clear as possible so a reviewer can follow the same steps. List what UI interactions are needed, or if there are automated tests then list what should be run. For example:

1. Go to: https://scout.tsdataclinic.com/explore/NYC
2. Advance through screen using Tab key to 1st dataset link.
3. Unable to see where I was on the page as there's no outline to indicate focus.
4. Added outline to dataset link to allow for visual tracking of focused element.

## Checklist Before Requesting a Review

- [ ] I have performed a self-review of my code
- [ ] My code follows the Style Guidelines and Best Practices outlined in the project wiki
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made changes to the documentation, if applicable
- [ ] My change generates no new warnings or failed tests
- [ ] If it is a core feature, I have added thorough tests
- [ ] I have implemented analytics, if applicable

### CODE REVIEW

File: packages/frontend/src/components/Dataset/Dataset.scss [hunk 1]
```diff
@@ -1,6 +1,11 @@
 .dataset {
   display: grid;
 
+  .data-link-title{
+    &:focus-visible{
```
Comment (jps327) [inline packages/frontend/src/components/Dataset/Dataset.scss:5]: Remember to keep your code following any patterns you already see in the codebase you're editing. In this case, remember to add a space before the opening curly braces.

File: packages/frontend/src/components/Dataset/Dataset.scss [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+      outline:3px solid #000;
```
Comment (jps327) [inline packages/frontend/src/components/Dataset/Dataset.scss:6]: Similar to the last comment, remember to stick to existing code conventions. In this case, adding a space in between the colon and the 3px

**Review by jps327 — CHANGES_REQUESTED** [review]
Just some minor edits!

**Comment by Detroit-the-Dev** [comment]
Hey Pablo!

Fixed the spacing to be in conformance with codebase conventions.

Thanks!

**Review by jps327 — APPROVED** [review]
Thanks!

**Pull request merged**